src/elements.js

```diff
@@ -96,9 +97,7 @@ function getOrCreateElement(elements, index, type, initProperties) {
   let element = elements[index];
   if (!element || !(element instanceof elementClass)) {
     element = elements[index] = new elementClass();
-    if (isObject(initProperties)) {
-      Object.assign(element, initProperties);
-    }
+    applyInitProperties(element, initProperties);
```

Comment:
Looks like `Object.assign` handles `undefined`, so I think the `isObject` check can be removed